### ISSUE TITLE
Segregate graphics and non-graphics tests

### ISSUE THREAD

**Issue by Ilia-Kosenkov** [extendr/extendr#378]
Currently, graphics tests produce random memory access errors, crashing test suites. 
A recent example would be #377 https://github.com/extendr/extendr/runs/5206578831?check_suite_focus=true#step:11:121
Crashes are observed on both Windows and Linux machines, but because the number of Linux instances we spin is larger than any other systems, the majority of recent failures originate from Linux runners.

I proposed (somewhere, can't find the link) that we separate graphics tests from all other tests, i.e. remove it from `tests-all` feature.
This will allow us to run regular tests in parallel (as we do now) and then run only graphics tests with `--test-threads 1`. This will probably help us to diagnose graphics problems more easily.

TODO:
- [x] Finish #373
- [ ] Ensure `graphics` is not part of the `tests-all` set and is not tested 'by default' so this CI step does not test graphics https://github.com/extendr/extendr/blob/678f02c51573563be60969691612a9c4720123aa/.github/workflows/test.yml#L157
- [ ] Modify CI to execute additional `cargo test --features graphics -- --test-threads 1` 
- [ ] Remove `--test-threads 1` from remaining tests https://github.com/extendr/extendr/blob/678f02c51573563be60969691612a9c4720123aa/.github/workflows/test.yml#L417-L421

I am mentioning and assigning it to @yutannihilation. I consider this issue to be a priority as it affects and complicates the process of code review.

**Comment by yutannihilation** [comment]
> I proposed (somewhere, can't find the link) that we separate graphics tests from all other tests

Sounds good. I might have missed your suggestion, sorry.

I think `tests-all` should contain `graphics` because it's `all`. So, probably we should create another one e.g. `tests-regular`? Anyway, it's a trivial thing. I agree with the general direction.
